src/patternfly/components/Divider/examples/Divider.md

```diff
@@ -33,3 +56,5 @@ The divider renders as an `<hr>` by default. It is possible to make the divider
 | Class | Applied to | Outcome |
 | -- | -- | -- |
 | `.pf-c-divider` | `<hr>`, `<li>`, `<div>` | Defines the divider component. |
+| `.pf-m-vertical` | `.pf-c-divider` | Modifies the divider component from horizontal to vertical. This modifier requires that the parent has an explicit or implicit height, or has a flex or grid based layout parent. |
+| `.pf-m-inset-{none, sm, md, lg, xl, 2xl, 3xl}{-on-[sm, md, lg, xl, 2xl]}` | `.pf-c-divider` | Modifies a divider inset to match spacer system. |
```

Comment:
Can you explain further what inset is here, I think it might help to include more info for designers/devs here

Reply:
And I'm a little confused too

Reply:
@christiemolloy updated